src/test/java/com/arangodb/ArangoCollectionTest.java

```diff
@@ -2612,14 +2613,47 @@ public void getPropeties() {
     public void changeProperties() {
         final CollectionPropertiesEntity properties = collection.getProperties();
         assertThat(properties.getWaitForSync(), is(notNullValue()));
+        if (isAtLeastVersion(3, 7)) {
+            assertThat(properties.getSchema(), is(nullValue()));
+        }
 
-        final CollectionPropertiesEntity changedProperties = collection.changeProperties(new CollectionPropertiesOptions()
-                .waitForSync(!properties.getWaitForSync()));
+        String schemaRule = ("{  " +
+                "           \"properties\": {" +
+                "               \"number\": {" +
+                "                   \"type\": \"number\"" +
+                "               }" +
+                "           }" +
+                "       }")
+                .replaceAll("\\s", "");
+        String schemaMessage = "The document has problems!";
```

Comment:
Not necessarily in this pr, but shouldn't the message be configurable?

Reply:
This code here is from the tests. Also, the message is the same for any validation error, since:
> The schema validation cannot pin-point which part of a rule made it fail 

See https://www.arangodb.com/docs/stable/data-modeling-documents-schema-validation.html#error-message